Source/OxyPlot/Series/StairStepSeries.cs

```diff
@@ -156,47 +156,53 @@ public override void Render(IRenderContext rc)
                                               : this.VerticalStrokeThickness;
 
             var actualColor = this.GetSelectableColor(this.ActualColor);
+            var edgeRenderingMode = this.EdgeRenderingMode.GetActual(EdgeRenderingMode.PreferSharpness);
 
-            Action<IList<ScreenPoint>, IList<ScreenPoint>> renderPoints = (lpts, mpts) =>
+            Action<IList<ScreenPoint>, IList<ScreenPoint>> renderPoints = (linePoints, markerPoints) =>
                 {
-                    // clip the line segments with the clipping rectangle
                     if (this.StrokeThickness > 0 && lineStyle != LineStyle.None)
                     {
+                        // TODO: Need ActualVerticalLineStyle; without it, when VerticalLineStyle is
```

Comment:
What's this comment about? Is this an outstanding issue?

Reply:
Yes, though not one that's been reported, as far as I'm aware.  It's just an additional opportunity for performance improvement that I wanted to call out, but one that I wasn't planning to tackle.